docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "How-To: Upgrade Spryker instance to Marketplace"
+description: This article provides details how to upgrade Spryker instance to Marketplace
+template: howto-guide-template
+---
+
+This articles provides details how to upgrade existing instance of Spryker shop to Marketplace
```

Comment:
Note for the feature please use text validation tools, e.g. Grammarly
```suggestion
This article provides details on how to upgrade the existing instance of Spryker Shop to Marketplace
```

Reply:
fixed